lib/components/home/artist_rails/artist_rail.js

```diff
@@ -23,7 +23,7 @@ const Animation = {
 
 class ArtistRail extends React.Component {
   state: {
-    artists: [Object],
+    artists: any[],
```

Comment:
Above is an array with an object, below is a type of array that contains anything. One is an instance, the other is a class 👍